core/commands/keybase/keybase.go

```diff
@@ -0,0 +1,37 @@
+package keybase
```

Comment:
Maybe `keyencode` is better here as it's the name of the name struct.

Also, `keybase` may be confusing, especially with a project not so far removed from our space having that name.